online-shop/src/app/components/presentational/products-details-view/products-details-view.component.html

```diff
@@ -0,0 +1,47 @@
+<html>
+<head>
+    <title>Product details</title>
+</head>
+<body>
+    <div class="body-component">
+        <div class="title-flex-container">
+            <h1>Product Name: {{ product?.productName }}</h1>
```

Comment:
If you want to get rid of the question marks here you can just add a `*ngIf="product"` at a higher level (e.g., first div).